tests/decompiler/index.ts

```diff
@@ -0,0 +1,51 @@
+import "allocator/arena";
+
+declare function println(s: i32):void;
+
+export class Test {
+  constructor(public i: i32){
```

Comment:
This doesn't decompile correctly.  Still need to figure out how to handle if's that have a result value.